docs/DeveloperGuide.md

```diff
@@ -285,29 +285,65 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC01 - Delete a member**
 
 **MSS**
```

Comment:
What do you think about adding a `guarantees` in between here?
```suggestion
**Use case: UC01 - Delete a member**

**Guarantees: MSS -> Specified event will be deleted.**

**MSS**
```

Reply:
Sounds great, I will implement that